Description: Incorporate upstream bugfixes to this version.
 Upstream tends to issue patches to fix bugs rather than a new
 source Tarball
Forwarded: not-needed
--- a/src/execline.c
+++ b/src/execline.c
@@ -459,6 +459,7 @@
 /* go until we hit an empty line, or reach end of proc.. */
 for( iline = txtstartline; iline <= procstop ; iline++ ) {
 	line = PLL.procline[ iline ];
+	if( line == NULL ) break;  /* stmt added scg 1/7/2014 ... bug fix, multiline attribute followed immed by #proc (no blank line) causes instability */
 	for( i = 0, emptyline = 1; line[i] != '\0'; i++ ) if( !isspace( (int) line[i] )) { emptyline = 0; break; }
 	if( emptyline ) break;
 	if( mode[0] == 'g' ) txtlen += (strlen( &line[i] ) + 2);  /* mode = "get", accumulate length sans leading ws */
--- a/src/init.c
+++ b/src/init.c
@@ -160,7 +160,7 @@
 	else if( Edev == 'g' ) {
 		/* terminate existing image and start a new one with new size.. */
 		Esetwinscale( (int)(pagewidth*Epixelsinch), (int)(pageheight*Epixelsinch), pagewidth, pageheight );
-		PLGG_setup( "", Epixelsinch, pagewidth, pageheight, uplefttx, upleftty );
+		PLGG_setup( "", Epixelsinch, pagewidth, pageheight, uplefttx, upleftty, maxdrivervect );
 		}
 #endif
 	}
